File: service/src/main/java/ru/diasoft/micro/dofacade/mock/UploadAttachmentRes.java
package ru.diasoft.micro.dofacade.mock;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UploadAttachmentRes {

    @JsonProperty("fileId")
    String fileId = "fileId";
    @JsonProperty("fileInfoId")
    Integer fileInfoId = 65454;
    @JsonProperty("fileName")
    String fileName = "fileName";
    @JsonProperty("fileSize")
    Integer fileSize = 89454;
    @JsonProperty("fileType")
    String fileType = "fileType";
    @JsonProperty("saveType")
    String saveType = "saveType";
    @JsonProperty("tempKey")
    String tempKey = "tempKey";

}

File: src/main/java/ru/diasoft/micro/dofacade/json/attachment/UploadAttachmentRes.kt
package ru.diasoft.micro.dofacade.json.attachment

import com.fasterxml.jackson.annotation.JsonProperty


data class UploadAttachmentRes(

        @JsonProperty("fileId")
        val fileId: String = "",
        @JsonProperty("fileInfoId")
        val fileInfoId: Int? = null,
        @JsonProperty("fileName")
        val fileName: String = "",
        @JsonProperty("fileSize")
        val fileSize: Int = 0,
        @JsonProperty("fileType")
        val fileType: String = "",
        @JsonProperty("saveType")
        val saveType: String = "",
        @JsonProperty("tempKey")
        val tempKey: String = ""
)
